packages/components/src/Disclosure/Disclosure.test.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { cleanup, fireEvent, render, waitFor } from "@testing-library/react";
+import { Disclosure } from ".";
+
+afterEach(cleanup);
+
+const sampleProps = {
+  title: "Example Disclosure Title",
+  open: false,
+};
+
+it("renders a Disclosure", () => {
+  const tree = renderer
+    .create(
+      <Disclosure {...sampleProps}>
+        <p>Wafer topping soufflé bear claw cake chocolate toffee.</p>
+      </Disclosure>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
+
+it("renders a Disclosure that is opened by default", () => {
+  const newProps = {
+    ...sampleProps,
+    defaultOpen: true,
+  };
```

Comment:
Not a big deal at all but I think I like the way you are splatting in your props in the `should not attempt to call the handler when it is not set` test a bit better.